Admin.js

```diff
@@ -11,7 +11,7 @@
 
 var Admin = {
 
```

Comment:
- eslint: All 'var' declarations must be at the top of the function scope.
- eslint: Unexpected var, use let or const instead.
- eslint: Strings must use doublequote.